makedoc.g

```diff
@@ -0,0 +1,33 @@
+##  makedoc.g for the KBMag package
+##  this creates the package documentation  
+##  needs: GAPDoc and AutoDoc packages, latex, pdflatex, mkindex 
+##  call this with GAP from within the package root directory 
+##  
+LoadPackage( "GAPDoc" );
+
+AutoDoc( rec( 
```

Comment:
This file does not actually work, as it fails to load the `AutoDoc` package. To fix this, just replace GAPDoc above by AutoDoc. (There is no reason to load GAPDoc, it is always loaded)

Reply:
Done